src/kotoba/agent/generator/genSquareVisitor

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Author: Daniele Fognini, Andreas Wuerl
+# SPDX-FileCopyrightText: © 2013-2014 Siemens AG
+
+# SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Is this suppose to be here? If this is picked from some other agent, Try to reuse it